src/add/operator/buffer.ts

```diff
@@ -4,4 +4,8 @@ import {buffer} from '../../operator/buffer';
 
 Observable.prototype.buffer = buffer;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  interface Observable<T> {
+    buffer: (closingNotifier: Observable<any>) => Observable<T[]>;
```

Comment:
Yeah, exactly. For now, is it okay to merge this as is, then start referencing David's interfaces as he adds them?